airflow/cli/cli_parser.py

```diff
@@ -339,6 +339,11 @@ def string_lower_type(val):
     help=("if set, the backfill will keep going even if some of the tasks failed"),
     action="store_true",
 )
+ARG_DISABLE_RETRY = Arg(
+    ("-dr", "--disable-retry"),
```

Comment:
The `-dr` flag feels unnecessary, and I don’t think we have a precedence for multi-character short options.

```suggestion
    ("--disable-retry",),
```